suites/apis/dbgapTest.js

```diff
@@ -0,0 +1,280 @@
+    let phst000179gsFileContents = await fence.do.getFileFromSignedUrlRes(
+      signedUrlgsPhs000179Res);
+
+    chai.expect(phs000178s3FileContents,
+      `User ${users.mainAcct.username} with access could NOT create s3 signed urls ` +
+      'and read file for a record in authorized dbGaP project phs000178'
+    ).to.equal(fence.props.awsBucketInfo.cdis_presigned_url_test.testdata);
+    chai.expect(phs000178gsFileContents,
+      `User ${users.mainAcct.username} with access could NOT create gs signed urls ` +
+      'and read file for a record in authorized dbGaP project phs000178'
+    ).to.equal(fence.props.googleBucketInfo.test.fileContents);
+
+    chai.expect(phst000179s3FileContents,
+      `User ${users.mainAcct.username} WITHOUT access COULD create s3 signed urls ` +
+      'and read file for a record in unauthorized dbGaP project phs000179'
+    ).to.not.equal(fence.props.awsBucketInfo.cdis_presigned_url_test.testdata);
+    chai.expect(phst000179gsFileContents,
+      `User ${users.mainAcct.username} WITHOUT access COULD create gs signed urls ` +
+      'and read file for a record in unauthorized dbGaP project phs000179'
+    ).to.not.equal(fence.props.googleBucketInfo.QA.fileContents);
+});
+
+Scenario('dbGaP + user.yaml Sync: ensure combined access @dbgapSyncing',
+  async (fence, indexd, users, files) => {
+    console.log('Running usersync job and adding dbgap sync to yaml sync');
+    bash.runJob('usersync', args='ADD_DBGAP true FORCE true');
+
+    // users.mainAcct has access to phs000178 through dbgap
+    console.log('Use mainAcct to create s3 signed URL for file phs000178')
+    const signedUrls3phs000178Res = await fence.do.createSignedUrl(
+      indexed_files.phs000178File.did, ['protocol=s3'],
+      users.mainAcct.accessTokenHeader
+    );
+
+    // users.mainAcct has access to QA through user.yaml
+    console.log('Use mainAcct to create s3 signed URL for file QA')
+    const signedUrlsQARes = await fence.do.createSignedUrl(
+      indexed_files.QAFile.did, ['protocol=s3'],
+      users.mainAcct.accessTokenHeader
+    );
+
+    let phs000178s3FileContents = await fence.do.getFileFromSignedUrlRes(
+      signedUrls3phs000178Res);
+    let QAFileContents = await fence.do.getFileFromSignedUrlRes(
+      signedUrlsQARes);
+
+    chai.expect(phs000178s3FileContents,
+      `User ${users.mainAcct.username} with access could NOT create s3 signed urls ` +
+      'and read file for a record in authorized dbGaP project phs000178'
+    ).to.equal(fence.props.awsBucketInfo.cdis_presigned_url_test.testdata);
+    chai.expect(QAFileContents,
+      `User ${users.mainAcct.username} with access could NOT create s3 signed urls ` +
+      'and read file for a record in authorized program QA'
+    ).to.equal(fence.props.awsBucketInfo.cdis_presigned_url_test.testdata);
+});
+
+Scenario('dbGaP + user.yaml Sync (from prev test): ensure user cannot create/update/delete dbGaP indexd records @dbgapSyncing',
```

Comment:
```suggestion
Scenario('dbGaP + user.yaml Sync (from prev test): ensure user without dbGap access cannot create/update/delete dbGaP indexd records @dbgapSyncing',
```

Reply:
maybe add a test `ensure user with dbGap access can create/update/delete dbGaP indexd records`

Reply:
well, so that's sorta the weird part. We actually don't want just anyone with access to be able to modify the indexd record. We'd have to give special access to data nodes who need to create records for a specific project, but anyone with access should only have read

Reply:
then do you think a `ensure user with dbGap read access cannot create/update/delete dbGaP indexd records` test (using mainAcct) would be useful? maybe both or maybe in replacement of this one

Reply:
yeah, I could update this test to check user with _and_ without dbgap access, since in both cases they shouldn't be able to create/update/delete